.ci/setup_env_alinux.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || "source /usr/lib/os-release"
+source "${cidir}/lib.sh"
+
+# Obtain AlibabaCloud Linux version
+# Either /etc/os-release or /usr/lib/os-release is source´ed
+# so that VERSION_ID is already exported.
+alinux_version=$(grep VERSION_ID /etc/os-release | cut -d '"' -f2)
```

Comment:
I think you can just reference `$VERSION_ID` as you've already sourced that file?

Reply:
updated